### ISSUE TITLE
Pl update mavlink module

### ISSUE THREAD

**Issue by olliw42** [mavlink/qgroundcontrol#8105]
May I kindly ask to update the mavlink module?

This would help with things like PR https://github.com/mavlink/qgroundcontrol/pull/8007.

THX :)

**Comment by DonLakeFlyer** [comment]
Anybody can update to mavlink head. Just do a pull in the submodule dir and submit a pull request.

**Comment by olliw42** [comment]
thx for the quick response.

I hear your words but am not sure I really understand. Git submodules and how to handle them properly have remained a mystery to me. Sorry for my limitations. 

I'm worried I rather would create more problems than helping resolve. So, in trying to make sense of your words I did this:

created a branch mavupdate-01
cd'ed to libs/mavlink/include/mavlink/v2.0, showing me (6b8bb7c...)
entering git pull throws an error "You are not currently on a branch", so I did:
git checkout master
git pull 
cd'ed back
commited
pushed branch

which results in PR https://github.com/mavlink/qgroundcontrol/pull/8106. Pl carefully double check if this PR does what it's supposed to do.
